src/Umbraco.Forms.Integrations.Crm.Hubspot/Configuration/ServiceConfiguration.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Umbraco.Forms.Integrations.Crm.Hubspot.Configuration
+{
+    public static class ServiceConfiguration
+    {
+        /// <summary>
+        /// Integrated services with their token URIs
+        /// </summary>
+        public static Dictionary<string, string> ServiceProviders = new()
+        {
+            { "Hubspot", "oauth/v1/token" }, 
+            { "HubspotForms", "oauth/v1/token" }, 
+            { "Semrush", "oauth2/access_token" }, 
```

Comment:
Again references to other services here shouldn't be part of the HubSpot integration.